client/src/LogoutButton.jsx

```diff
@@ -1,14 +1,13 @@
 import React from "react";
 
-import AjaxHelper from "./AjaxHelper";
+import { logoutUrl } from "./routes";
 
 const LogoutButton = () => {
-    const ajax = AjaxHelper.getInstance();
     const clearToken = () => {
         sessionStorage.removeItem("token");
     };
     return (
-        <a href={ajax.logoutUrl} className="button" onClick={clearToken}>
+        <a href={logoutUrl} className="button" onClick={clearToken}>
```

Comment:
Out of scope for this PR, but for the to-do list: we shouldn't display the log out button when we don't have a token; maybe we actually need a log in button that can trigger the log in process manually.